email.go

```diff
@@ -21,32 +31,22 @@ type Email struct {
 	body    string
 }
 
-func NewEmail(smtpHost string, from string, to []string, subject, body string) *Email {
-	return &Email{
-		host:    smtpHost,
-		from:    from,
-		to:      to,
-		subject: subject,
-		body:    body,
-	}
-}
-
-func (r *Email) SendEmail() (bool, error) {
+func (r *EmailClient) Send(to []string, subject, body string) error {
 
 	m := gomail.NewMessage()
-	m.SetHeader("From", r.from)
+	m.SetHeader("From", r.fromAddress)
 	m.SetHeader("mailed-by", "cyverse.org")
-	m.SetHeader("To", r.to[0])
-	m.SetHeader("Subject", r.subject)
-	m.SetBody("text/html", r.body)
+	m.SetHeader("To", to[0])
+	m.SetHeader("Subject", subject)
+	m.SetBody("text/html", body)
 
-	d := gomail.Dialer{Host: r.host, Port: 25}
+	d := gomail.Dialer{Host: r.smtpHost, Port: r.smtpPort}
 
 	// Send the email to Bob, Cora and Dan.
```

Comment:
It looks like this comment was copied from an example. You can probably remove it.